jiracli/templates.go

```diff
@@ -293,14 +293,30 @@ const defaultDebugTemplate = "{{ . | toJson}}\n"
 const defaultListTemplate = "{{ range .issues }}{{ .key | append \":\" | printf \"%-12s\"}} {{ .fields.summary }}\n{{ end }}"
 
 const defaultTableTemplate = `{{/* table template */ -}}
-{{$w := sub termWidth 107 -}}
-+{{ "-" | rep 16 }}+{{ "-" | rep $w }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+{{ "-" | rep 12 }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+
-| {{ "Issue" | printf "%-14s" }} | {{ "Summary" | printf (printf "%%-%ds" (sub $w 2)) }} | {{ "Type" | printf "%-12s"}} | {{ "Priority" | printf "%-12s" }} | {{ "Status" | printf "%-12s" }} | {{ "Age" | printf "%-10s" }} | {{ "Reporter" | printf "%-12s" }} | {{ "Assignee" | printf "%-12s" }} |
-+{{ "-" | rep 16 }}+{{ "-" | rep $w }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+{{ "-" | rep 12 }}+{{ "-" | rep 14 }}+{{ "-" | rep 14 }}+
+{{$issuew := int 10 -}}
+{{$typew := int $issuew -}}
+{{$priorityw := int 14 -}}
+{{$statusw := $priorityw -}}
+{{$agew := int 10 -}}
+{{$reporterw := int 25 -}}
+{{$assigneew := int 12 -}}
+{{$dissuew := int (add 2 $issuew) -}}
```

Comment:
`add` seems to return `int64`, but the `sub` template function only wants `int` type args, so there's just a ton of casting going on.

Something like `{{$newval := sub $bar (add 2 $foo) -}}` would give an error like `ERROR Invalid Usage: template: gojira:1:30: executing "gojira" at <$ifoo>: wrong type for value; expected int; got int64`